packages/dds/merge-tree/src/ops.ts

```diff
@@ -102,6 +102,10 @@ export interface IMergeTreeRemoveMsg extends IMergeTreeDelta {
 	relativePos2?: IRelativePosition;
 }
 
+/**
+ * @deprecated This functionality lacks test coverage and is largely unused. It
```

Comment:
For all of the newly deprecated items here, what should consumers use instead? We should document that explicitly. If there are no recommended replacements, I would recommend noting that explicitly as well.

Reply:
Also nit: I'm not sure calling out that the functionality is untested is relevant to consumers. We could probably just omit that part.

Reply:
I like calling out that the code is untested as it gives some justification and context behind its removal. That said, I agree the fact that there's no replacement could be made a bit more explicit and in rewording the message I removed that bit.